app/src/main/res/drawable/ic_settings_fingerprint.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <group>
+    <clip-path
+        android:pathData="M8,0L16,0A8,8 0,0 1,24 8L24,16A8,8 0,0 1,16 24L8,24A8,8 0,0 1,0 16L0,8A8,8 0,0 1,8 0z"/>
+    <path
+        android:pathData="M0,0h24v24h-24z"
+        android:fillColor="#FF518F"/>
```

Comment:
use colors from theme

Reply:
looks like overhead, we can tint icons in the places of usage, and icon is a icon - it's already ready for use and we don't manipulate colours in icons

Reply:
No, it's not. 
Once we will add the night theme, some colors will automatically changed and we need all icons to be changed as well